src/utils/simpleSvgPlaceholder.ts

```diff
@@ -0,0 +1,74 @@
+/*
+MIT License
+
+Copyright (c) 2020 Cloud Four
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+https://github.com/cloudfour/simple-svg-placeholder
+*/
+
+interface PlaceholderImageOptions {
+  width: number;
+  height: number;
+  text?: string;
+  fontFamily?: string;
+  fontWeight?: string;
+  fontSize?: number;
+  dy?: number;
+  bgColor?: string;
+  textColor?: string;
+  dataUri?: boolean;
+  charset?: string;
+}
+
+export default function simpleSvgPlaceholder({
```

Comment:
Nitpick: this could be renamed or exported as another const to something simpler (e.g., `imagePlaceholder`)since the license has the source info.